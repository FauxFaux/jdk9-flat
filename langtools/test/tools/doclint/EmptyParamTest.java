/*
 * @test /nodynamiccopyright/
 * @build DocLintTester
 * @run main DocLintTester -Xmsgs:-syntax EmptyParamTest.java
 * @run main DocLintTester -Xmsgs:syntax -ref EmptyParamTest.out EmptyParamTest.java
 */

/** . */
public class EmptyParamTest {
    /** @param i */
    int emptyParam(int i) { }
}
