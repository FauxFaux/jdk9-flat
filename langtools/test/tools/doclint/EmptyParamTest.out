EmptyParamTest.java:10: warning: no description for @param
    /** @param i */
        ^
1 warning

